CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ The development prefers [Python 3.11](https://docs.python.org/3/whatsnew/3.11.ht
 
 ```bash
 # clone your fork
-git clone https://github.com/williamcaban/lightspeed-service.git
+git clone https://github.com/YOUR-GIT-PROFILE/lightspeed-service.git
```

Comment:
I'll try my luck with this not being separate PR. 😃

Reply:
i'll allow it :)